### PR TITLE
Add fhir service tests

### PR DESCRIPTION
**Pull request by ThishaniLucas**
Adding tests with 72.76% coverage.

### CODE REVIEW

File: fhirr4/ballerina/src/main/resources/fhirservice/Ballerina.toml [hunk 1]
```diff
@@ -20,6 +20,11 @@ org = "ballerinax"
 name = "health.fhir.r4.parser"
 version = "${parser.version}"
 
+[[dependency]]
+org = "ballerinax"
+name = "health.fhir.r4.international401"
+version = "${international401.version}"
```
Comment (sameeragunarathne) [inline fhirr4/ballerina/src/main/resources/fhirservice/Ballerina.toml:26]: Will this be a issue when releasing fhir service? Instead can't we create a simple test resource record and use it with a test config for the tests?

Comment (ThishaniLucas) [reply fhirr4/ballerina/src/main/resources/fhirservice/Ballerina.toml:26]: It wouldn't be an issue I suppose. We can just use whatever the released version of international401. There won't be any cyclic dependencies or so.

File: fhirr4/ballerina/src/main/resources/fhirservice/tests/test.bal [hunk 2]
```diff
@@ -0,0 +1,276 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
```
Comment (nirmal070125) [inline fhirr4/ballerina/src/main/resources/fhirservice/tests/test.bal:1]: Let's rename to test_fhir_service.bal or something appropriate.

Comment (ThishaniLucas) [reply fhirr4/ballerina/src/main/resources/fhirservice/tests/test.bal]: Renamed to fhir_service_test.bal.

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**